src/plugins/validation.ts

```diff
@@ -265,51 +271,60 @@ export class ValidationPlugin {
     doesFileExist: boolean,
     SNOWError?: string,
     statusCode?: number,
+    restrictNames?:boolean,
   ):((v: string) => string | true | undefined) => {
     return () => {
-      const fileName = file.name.length>20 
-        ? file.name.substring(0, 12) + '...' 
+      const fileName = file.name.length>20
+        ? file.name.substring(0, 12) + '...'
             + file.name.substring(file.name.length-8, file.name.length)
         : file.name;
       const fileSize = file.size;
       const isValidExtension = validExtensions.some((ext)=>
         fileName.substring(fileName.lastIndexOf(".")+1).toLowerCase() === ext
       )
+      const restrictedNames = ["DescriptionOfWork.docx",
+        "IncrementalFundingPlan.docx",
+        "RequirementsChecklist.docx",
+        "IGCE.xlsx",
+        "EvaluationPlan.docx",]
       if (!isValidExtension){
         return `'${fileName}' is not a valid format or has been corrupted. ` +
                 `Please upload a valid .${validExtensions.slice(0, -1).join(", .")} or ` +
                 `.${validExtensions.slice(-1)} file.`
       }
+      // list of names that a file can not have
+      else if(restrictNames && restrictedNames.includes(file.name)){

```

Comment:
1. remove `else`
2. with restricted names as a parameter, change to if `restrictedNames.length && restrictedNames.includes(file.name)`